hist/hist/src/TH1.cxx

```diff
@@ -4552,7 +4552,7 @@ TVirtualHistPainter *TH1::GetPainter(Option_t *option)
 ///
 /// \param[in] n maximum size of array xp and size of array p (if given)
 /// \param[out] xp array to be filled with nq quantiles evaluated at (p). Memory has to be preallocated by caller.
-/// If p is null (default value), then xp is actually set to the (first n) histogram bin edges
```

Comment:
The comment is maybe not correct. 
xp is always set to nbin+1 value, even if n is less than nbin+1

Reply:
This simple tests shows the opposite: https://github.com/root-project/root/pull/16782#discussion_r1820978143